cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -210,6 +212,10 @@ export default {
     cy.do([dropdownButton.click(), DropdownMenu().find(Button()).click()]);
   },
 
+  goToItemDetails(barcode) {
+    cy.do(Link(including(barcode)).click());
+  },
+
```

Comment:
We already have a method for this, please use:
`SearchPane.findResultRowIndexByContent(content).then((rowIndex) => { SearchResults.clickOnCell(content, row );});`